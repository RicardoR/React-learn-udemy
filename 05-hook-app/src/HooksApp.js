import React from "react"

export const HookApp = () => { 
    return (
        <div>Hello!</div>
    )
}