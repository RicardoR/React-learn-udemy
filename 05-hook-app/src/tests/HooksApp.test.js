import React from 'react';
import { HookApp } from '../HooksApp';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';

describe('<HooksApp /> UT', () => {
  test('should display Hello! in the title', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('h1').text()).toBe('Hello!');
  });
});
