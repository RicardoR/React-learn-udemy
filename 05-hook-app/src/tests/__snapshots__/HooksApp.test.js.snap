// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HooksApp /> UT should display Hello! in the title 1`] = `
<div>
  <h1>
    Hello!
  </h1>
</div>
`;
